doc/reference/reference_lua/box_ctl/set_on_shutdown_timeout.rst

```diff
@@ -0,0 +1,16 @@
+..  _box_ctl-on_shutdown_timeout:
+
+===============================================================================
+box.ctl.set_on_shutdown_timeout()
+===============================================================================
+
+..  module:: box.ctl
+
+..  function:: set_on_shutdown_timeout([timeout])
+
+     Set a timeout for the :ref:`on_shutdown <box_ctl-on_shutdown>` trigger.
```

Comment:
Would be nice to elaborate what happens if timeout occurs.

Reply:
Added info about timeout